test/unit/lib/whitehall/asset_manager_storage_test.rb

```diff
@@ -53,22 +53,6 @@ class Whitehall::AssetManagerStorage::FileTest < ActiveSupport::TestCase
     assert_equal "http://assets-host/media/asset_manager_id/sample.docx", model.file.url
   end
 
-  test "returns file url using asset_manager_id when the model has an asset variant" do
-    model = build(:attachment_data, attachable: build(:draft_edition, id: 1))
-    model.save!
-    model.reload
-
-    assert_equal "http://assets-host/media/asset_manager_id_thumbnail/thumbnail_greenpaper.pdf.png", model.file.url(:thumbnail)
-  end
-
-  test "returns nil when the model has assets but the requested variant is not available" do
```

Comment:
Bit of a confusing one, can't really find a way to rewrite it, and unsure it was doing much. OK to delete.

Info dump from hazy memories: When implementing the new asset IDs, as opposed to legacy paths, we needed a way to not send broken url for variants that may still be uploading, to publishing-api. What would have been nice, but we were unable to do, was to get the url method to just return `nil` when any of these asset variants is not available. And then presumably rerun the workers and patch that data on subsequent retries. 
I believe it was some carrierwave internal logic that causes the `path` value to be returned if `url` is nil, as per the test `returns store path when the model has no assets, although it should (still uploading or error has occurred)`. 
Whilst the ubiquitous method `all_asset_variants_uploaded?` is also used to cause some worker retries, if we could return a proper nil here, that might allow us to simplify the usage of that method. It is used heavily in presenters to ensure we don't send any half-broken attachment url data to publishing-api. Of course, famously failing in the `Person` same name file case.